src/screens/Home.js

```diff
@@ -50,12 +52,15 @@ export default ({
     synchronizeStorageToRedux();
     const categories = getAllCategories();
     setCategories(categories);
+    synchronizeStorageToRedux()
   }, []);
 
   const openCategoryPhrases = item => {
     const categoryId = item.id;
     setCurrentCategory(categoryId);
     // fetch Phrases for category
+    const phrasesForCategory = getPhrasesForCategoryId(item.id);
+    setPhrases(phrasesForCategory); 
```

Comment:
Conflict resolving mistake ?
`
const phrasesForCategory = getPhrasesForCategoryId(item.id);
    setPhrases(phrasesForCategory)
`
is duplicated